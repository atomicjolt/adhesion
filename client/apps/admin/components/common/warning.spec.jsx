import React from 'react';
import { shallow } from 'enzyme';
import Warning from './warning';

describe('warning', () => {
  let result;
  let props;

  beforeEach(() => {
    props = {
      text: 'bfcoder was here',
    };
    result = shallow(<Warning {...props} />);
  });

  it('matches the snapshot', () => {
    expect(result).toMatchSnapshot();
  });

  it('renders the warning', () => {
    const warning = result.find('div');
    expect(warning.props().children[1]).toContain(props.text);
  });
});
