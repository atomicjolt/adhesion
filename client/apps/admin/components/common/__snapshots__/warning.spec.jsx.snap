// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`warning matches the snapshot 1`] = `
<div
  style={
    Object {
      "color": "red",
    }
  }
>
  Warning! Bad JSON! 
  bfcoder was here
</div>
`;
